### PR TITLE
feat: support config session settings on values.yaml

### PR DESCRIPTION
**Pull request by ZhiHanZ**

### CODE REVIEW

**Review by hantmac — APPROVED** [review]

File: charts/databend-query/values.yaml [hunk 1]
```diff
@@ -93,14 +98,6 @@ config:
 
   # [meta]
   meta:
-    # If databend-meta is hosted in the same cluster as databend-data, you can enable this to generate endpoints in a K8s native way.
```
Comment (everpcpc) [inline charts/databend-query/values.yaml:96]: Why deleting these?

Comment (ZhiHanZ) [reply charts/databend-query/values.yaml]: oops... didn't notice the update patch, reverted those parts.

**Comment by everpcpc** [comment]
CI seems failed.

File: charts/databend-query/templates/configmap.yaml [hunk 2]
```diff
@@ -37,6 +37,14 @@ data:
         {{- end }}
       {{- end }}
 
+      [[query.settings]]
```
Comment (everpcpc) [inline charts/databend-query/templates/configmap.yaml:40]:
```suggestion
      [query.settings]
```

File: charts/databend-query/values.yaml [hunk 3]
```diff
@@ -114,14 +119,14 @@ config:
   # [storage]
   storage:
     type: "s3"
-
+    allow_insecure: true
```
Comment (everpcpc) [inline charts/databend-query/values.yaml:122]: This should not be default to true?

**Review by everpcpc — APPROVED** [review]

**Pull request merged**